models.py

```diff
@@ -0,0 +1,75 @@
+from django.db import models
+
+# Create your models here.
+class User(models.Model):	
+	name = models.CharField(max_length=50, unique=True)
+	emailid = models.EmailField()
+	contactnum = models.CharField(max_length=50)
+	address = models.CharField(max_length=200)
+
+class CustomEvent(models.Model):
+	event_id = models.ForeignKey(Event, on_delete=models.CASCADE)
+	name = models.CharField(max_length=50)
+	description = models.CharField(max_length=200)
+
+class Event(models.Model):
+	start_time = models.DateTimeField()
+	end_time = models.DateTimeField()
+	venue_id = models.ForeignKey(Venue, on_delete=models.CASCADE)
+	caterer_id = models.ForeignKey(Caterer, on_delete=models.CASCADE)
+	decorater_id = models.ForeignKey(Decorater, on_delete=models.CASCADE)
+	vendor_id = models.ForeignKey(Vendor, on_delete=models.CASCADE)
```

Comment:
Skeptical about requirement of vendor_id here. Maybe a mistake in ERD. Any justification that can be given?